packages/terra-tabs/src/common-tabs/_HiddenTab.jsx

```diff
@@ -153,18 +203,28 @@ const HiddenTab = ({
       {...attributes}
       id={id}
       aria-controls={associatedPanelId}
-      role="tab"
+      role={showAddButton ? 'button' : 'tab'}
```

Comment:
why do we need this role switch..??

Reply:
This role switch is to differentiate add button from the hidden tab. If it's an add button then the role is set to button.